File: src/main/java/com/github/alexandrecarlton/idea/settings/layout/editor/codestyle/java/MatchingRuleOrder.java
package com.github.alexandrecarlton.idea.settings.layout.editor.codestyle.java;

import com.fasterxml.jackson.annotation.JsonProperty;

public enum MatchingRuleOrder {

  @JsonProperty("keep order")
  KEEP_ORDER,

  @JsonProperty("order by name")
  ORDER_BY_NAME

}

File: src/main/java/com/github/alexandrecarlton/idea/settings/layout/editor/codestyle/java/arrangement/MatchingRuleOrder.kt
package com.github.alexandrecarlton.idea.settings.layout.editor.codestyle.java.arrangement

import com.fasterxml.jackson.annotation.JsonProperty

enum class MatchingRuleOrder {

    @JsonProperty("keep order")
    KEEP_ORDER,

    @JsonProperty("order by name")
    ORDER_BY_NAME
}
